src/textfont.ts

```diff
@@ -197,7 +198,7 @@ export class TextFont {
       selectedFont = new TextFont(candidates[0]);
     } else {
       const bold = TextFont.fontWeightToBold(fd.weight);
-      const italic = TextFont.fontStyleToItalic(fd.style);
+      const italic = TextFont.fontStyleToItalic(fd.weight);
```

Comment:
@0xfe is that right? is `style` the same as `weight` or the same as `family`?

I do not remember :)

Reply:
fixed, thank @AaronDavidNewman